ingest.go

```diff
@@ -24,12 +24,16 @@ func findSuites(nodes []xmlNode, suites chan Suite) {
 
 func ingestSuite(root xmlNode) Suite {
 	suite := Suite{
-		Name:    root.Attr("name"),
-		Package: root.Attr("package"),
+		Name:       root.Attr("name"),
+		Package:    root.Attr("package"),
+		Properties: root.Attrs,
```

Comment:
Is there something in the phpunit file that would populate these properties? I didn't see any in the xml, and this change differs from https://github.com/Bornholm/go-junit/commit/ca252901f512fbde7c4ade03ca74fa0811b6e1ed#diff-b3dcaff765c26b1a3977bd264d094411R33.

Reply:
Check it here please testdata/phpunit.xml 
Most important for my task are fields "file" and "line". I'm not sure if it should be popped up into separate Suite member because it looks like phpunit-specific feature.